homeassistant/components/google_generative_ai_conversation/conversation.py

```diff
@@ -429,6 +429,14 @@ async def _async_handle_message(
                     raise HomeAssistantError(
                         f"The message got blocked due to content violations, reason: {chat_response.prompt_feedback.block_reason_message}"
                     )
+                if not chat_response.candidates:
+                    LOGGER.error(
+                        "No candidates found in the response: %s",
+                        chat_response,
+                    )
+                    raise HomeAssistantError(
+                        "Sorry, I had a problem getting a response from Google Generative AI."
```

Comment:
Extract this to a constant since it's used in multiple places.

Reply:
Done